one_fm/overrides/leave_application.py

```diff
@@ -286,41 +288,26 @@ def notify_leave_approver(self):
                     "total_leave_days": self.total_leave_days,
                     "workflow_state": self.workflow_state,
                     "posting_date": self.posting_date,
-                    "base_url": frappe.utils.get_url()
+                    "base_url": frappe.utils.get_url(),
```

Comment:
KIndly check this, and ascertain it returns the correct url, i once hadn an issue with it

Reply:
yes i tested it with the email notification it was taking to the required leave applicaion